00-RELEASENOTES

```diff
@@ -9,6 +9,22 @@ CRITICAL: There is a critical bug affecting MOST USERS. Upgrade ASAP.
 SECURITY: There are security fixes in the release.
 --------------------------------------------------------------------------------
 
+================================================================================
+Valkey 8.1.0 GA  -  Released Mon 31 March 2025
+================================================================================
+Upgrade urgency LOW: This is the first release of Valkey 8.1, 
+a minor version update designed to further enhance performance, reliability, observability and usability
+over Valkey 8.0 for all Valkey installations. This release is fully compatible with all previous Valkey releases 
+as well as Redis OSS 7.2.4.
```

Comment:
This part is OK for the 00-RELEASENOTES file, but for the github release, I think we should add explicit links to the rc1 and rc2 release notes here. On the download page on the website, we just link to the github release, which only shows short text but users also need to see the release notes of the release candidates.